noxfile.py

```diff
@@ -714,6 +683,9 @@ def manifest(session: nox.Session) -> None:
     positives can be ignored by adding file patterns and paths to
     `ignore` under `[tool.check-manifest]` in `pyproject.toml`.
     """
+    # check-manifest would be suitable as a pre-commit hook, except that
+    # it requires ∼10 seconds to build the package, which would triple
+    # the time needed to run pre-commit
```

Comment:
We could revisit this later on if we switch to using `uv build` if it is significantly faster.